src/Database/Schema/MysqlSchemaDialect.php

```diff
@@ -213,7 +213,7 @@ public function convertOptionsDescription(TableSchema $schema, array $row): void
      */
     public function describeOptions(string $tableName): array
     {
-        [$_, $name] = $this->splitTablename($tableName);
+        $name = $this->splitTablename($tableName)[1];
```

Comment:
No need to change but I believe `[, $name]` would have worked too, the way you had it earlier.